### ISSUE TITLE
[Bug]: Removing bottom navigation bar via ImageEditorCustomWidgets causes layout to skip when moving items

### ISSUE THREAD

**Issue by adamkoch** [hm21/pro_image_editor#69]
### Package Version

3.0.4

### Flutter Version

3.22.2

### Platforms

Android, iOS

### How to reproduce?

Not sure if this is the right way to remove the bottom navigation bar, but I am doing:
```
ProImageEditorConfigs(
  customWidgets: ImageEditorCustomWidgets(
    bottomNavigationBar: const SizedBox(),
  ),
),
```

This seems to work fine. However, when dragging items around after that. The layout shifts back to having the space for the bottom navigation bar (so everything shifts upward). The preview moves up and shrinks. And the "delete" drop target stays in the right place but the actual drop target itself seems to be offset upward as well.

Video demonstrating:

https://github.com/hm21/pro_image_editor/assets/869261/c7b20684-ace3-42e6-aee8-751b8756f829

**Comment by hm21** [comment]
Thanks for your bug report.

Yes, it's true, if you remove the appbar or bottombar you will run into trouble. In the [example](https://github.com/hm21/pro_image_editor/blob/stable/example/lib/pages/custom_appbar_bottombar_example.dart) I also added this comment “kBottomNavigationBarHeight is important that helperlines will work” but I forgot to add it to the documentation in the `customWidgets`.
However, since I released version `3.0.0`, is it maybe possible that I can change the code a bit that it will work. I will do some tests and give you later feedback here.
